zerver/lib/actions.py

```diff
@@ -5551,6 +5552,14 @@ def do_remove_realm_domain(realm_domain: RealmDomain, acting_user: Optional[User
     event = dict(type="realm_domains", op="remove", domain=domain)
     send_event(realm, event, active_user_ids(realm.id))
 
+def do_add_realm_playground(realm: Realm, playground_info: Dict[str, str]) -> int:
+    realm_playground = RealmPlayground(realm=realm, **playground_info)
+    realm_playground.full_clean()
+    realm_playground.save()
+    event = dict(type="realm_playground", op="add", realm_playground=playground_info)
```

Comment:
We may want to have these events just send the whole data set, rather than doing an add/remove system, because it's expected to be a consistently small data set, and that makes the client-side code simpler.